src/components/Tool/FixPhoneNumbers/ContactPhoneNumbers.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import { mdiDelete, mdiLock } from '@mdi/js';
+import { Icon } from '@mdi/react';
+import StarIcon from '@mui/icons-material/Star';
+import StarOutlineIcon from '@mui/icons-material/StarOutline';
+import {
+  Box,
+  FormControl,
+  FormHelperText,
+  Grid,
+  Hidden,
+  TextField,
+  Theme,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import clsx from 'clsx';
+import { Field } from 'formik';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from 'tss-react/mui';
+import { useLocale } from 'src/hooks/useLocale';
+import { dateFormatShort } from 'src/lib/intlFormat';
+import { isEditableSource, sourceToStr } from 'src/utils/sourceHelper';
+import theme from '../../../theme';
+import { PhoneNumber } from './Contact';
+import { PersonInvalidNumberFragment } from './GetInvalidPhoneNumbers.generated';
+
+const useStyles = makeStyles()((theme: Theme) => ({
+  responsiveBorder: {
+    [theme.breakpoints.down('xs')]: {
+      paddingBottom: theme.spacing(2),
+      borderBottom: `1px solid ${theme.palette.cruGrayMedium.main}`,
+    },
+  },
+  paddingX: {
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
+  },
+  paddingB2: {
+    paddingBottom: theme.spacing(1),
+  },
+  hoverHighlight: {
+    '&:hover': {
+      color: theme.palette.mpdxBlue.main,
+      cursor: 'pointer',
+    },
+  },
+  ContactIconContainer: {
+    margin: theme.spacing(0, 1),
+    width: theme.spacing(4),
+    height: theme.spacing(4),
+  },
+}));
+
+interface Props {
+  key: string;
+  index: number;
+  person: PersonInvalidNumberFragment;
+  phoneNumber: PhoneNumber;
+  handleChangePrimary: (personId: string, index: number) => void;
+  handleDeleteNumberOpen: (person: {
+    personId: string;
+    phoneNumber: PhoneNumber;
+  }) => void;
+  handleSingleConfirm: (
+    person: PersonInvalidNumberFragment,
+    numbers: PhoneNumber[],
+  ) => void;
+  setFieldValue: (field: string, value: string) => void;
+  errors: any;
+}
+
+export const ContactPhoneNumbers: React.FC<Props> = ({
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  key,
```

Comment:
This variable is not used

Reply:
you don't need to create a parameter for it. When adding key React knows what it is.